demo/gpu_acceleration/memory.py

```diff
@@ -1,51 +0,0 @@
-import xgboost as xgb
```

Comment:
@trivialfis I am curious to know, why this file was removed, is it not a correct demo?

Reply:
@aktech. It's done internally so users don't have to do it themselves.